python/pylibvw.cc

```diff
@@ -20,8 +20,7 @@
 #define VWDLL_EXPORTS
 #include "../vowpalwabbit/vwdll.h"
 
```

Comment:
feels heavy.

Reply:
Most of these usages are for `std::string`, bringing that into the global namespace is dangerous because of how ubiquitous the name string is.